lua/solarized-osaka/theme.lua

```diff
@@ -313,9 +313,25 @@ function M.setup()
     ["@module"] = { link = "Include" },
 
     -- tsx
-    ["@tag.tsx"] = { fg = c.green500 },
     ["@constructor.tsx"] = { fg = c.blue500 },
     ["@tag.delimiter.tsx"] = { fg = c.orange500 },
+    ["@tag.tsx"] = { fg = c.yellow500 },
+    ["@keyword.return.tsx"] = { fg = "#768900", style = "italic" },
```

Comment:
Can you use a color from the palette instead of hex colors directly?

Reply:
The palette colors I used were too dim.
The reason for using these was to give a slightly darker color compared to the rest of the statements.

![Screenshot from 2024-03-08 19-06-51](https://github.com/craftzdog/solarized-osaka.nvim/assets/13983258/4d6c43c8-3ef9-443b-89c4-b2967f9a2bea)

You can notice here that it is slightly darker for the italic statements. Using the colors from the palette does not provide this color as they are either too dim or too bright.
If it is necessary to use something from the palette then these can be removed altogether and only leave the italic styling for differentiation.

Reply:
can you add for vue too? don't just react

Reply:
![image](https://github.com/craftzdog/solarized-osaka.nvim/assets/69358107/f204a4fa-d408-4ae9-ac1e-7818a490da30)
![image](https://github.com/craftzdog/solarized-osaka.nvim/assets/69358107/20d584e5-45b8-43b7-881b-f821483fdc3b)

and markdown highlighting 
![image](https://github.com/craftzdog/solarized-osaka.nvim/assets/69358107/80826737-0815-4540-959e-beb7dced02c7)
![image](https://github.com/craftzdog/solarized-osaka.nvim/assets/69358107/adbc436c-3f7e-4e10-8409-837e254d3741)

Reply:
For the markdown, it works appropriately for me. 

![Screenshot from 2024-03-09 01-11-19](https://github.com/craftzdog/solarized-osaka.nvim/assets/13983258/cb29eb5c-1af6-40fd-b294-ec85b4341bc3)


![Screenshot from 2024-03-09 01-11-57](https://github.com/craftzdog/solarized-osaka.nvim/assets/13983258/364ce0f2-a666-4ded-a20a-2d5dfbadeb3d)

Maybe you do not have the conceal level for markdown properly configured.

I am also using [obsidian.nvim](https://github.com/epwalsh/obsidian.nvim) in all `.md` files as this provides me with the `Task` attribute being shown properly as you would see in actual editors so maybe look into that.

Still without even using obsidian.nvim I still have appropriate markdown highlights. They could be made better but that is something not suitable with the limited colors provided in this color scheme.

Maybe I will look into that in the future I do find it lackluster in markdown but still it is more than enough the only thing lacking would be different colors for all headings as compared to [tokyonight.nvim](https://github.com/folke/tokyonight.nvim) but the rest seems to be the same.

Reply:
@craftzdog Could you please check if the `.md` files have these highlights mentioned here applying properly for you or do the markdown files look the same as they do for us? you can see how it appears for us above.

![Screenshot from 2024-03-09 01-51-07](https://github.com/craftzdog/solarized-osaka.nvim/assets/13983258/997c7a2a-576b-4d33-9040-b014f842de48)

- This should provide different highlights for the markdown h1 and h2 at least but that does not seem to be the case for both me and @bobbyrahmanda13.
- Most of these mentioned here in the theme do not seem to apply anything to markdown.
- I tested changing the color values here but they do not seem to be doing anything.

Reply:
yes, I like it, really like it for vue it looks better than before, for the markdown you provided it is very good I also like it but can you add a little highlight to the markdown, I beg you,

Vue has nuxt content with a component function for markdown so you can add styles like in this picture
![image](https://github.com/craftzdog/solarized-osaka.nvim/assets/69358107/45653a8a-e1cc-441c-8fba-1a9bb6a5b3ae)

because before it was only white
![image](https://github.com/craftzdog/solarized-osaka.nvim/assets/69358107/ee72d4b5-5cb1-4d80-b7eb-2118bb1c74fb)

Reply:
if it can be done then vue users if using this colorscheme will be amazed, because I don't see any other colorscheme working for this, if it can be done, if it can't, that's okay, I would still like the changes that happened to this colorscheme

Reply:
@bobbyrahmanda13 Could you create a public repository with this markdown being set with Vue and Nuxt so I can see how it works I have never worked with Nuxt or Vue.

- Also as this uses a different syntax altogether compared to standard markdown it would require its own treesitter parser or at least a module that would integrate this additional information inside markdown files which are not part of the standard markdown syntax.

- They provide a VScode [extension](https://marketplace.visualstudio.com/items?itemName=Nuxt.mdc) that provides the syntax for the MDC components which would use this syntax and currently in Neovim this does not seem to have any support so this is not something that can be configured by a color scheme as far as I think.

- I am not an expert myself but that is what I understand from this as they provide only Volar as the lsp configuration and there is no Treesitter parser for MDC components.

Link me a public repo with all this setup and I will see how MDC component parsing might be handled in this.

Keep in mind that this is completely out of scope for this PR.

Reply:
sorry, I think it can be done in this PR, maybe I will review it better, I am also not an expert in Vue and currently still learning, thanks for the advice

Reply:
> sorry, I think it can be done in this PR, maybe I will review it better, I am also not an expert in Vue and currently still learning, thanks for the advice

No worries these constraints come from things that are unique to Neovim compared to VScode and people usually create plugins for such features exclusively for VScode being the more popular editor.

If the feature is something more in demand it will eventually get its parser for treesitter but seems highly unlikely as they are using standard format `.md` with their own injections inside.

Reply:
> The palette colors I used were too dim.

The palette is constraints and I would like you to follow them. I can't allow hex colors.